frontend/src/individual_certificate.tsx

```diff
@@ -193,6 +195,10 @@ export function IndividualCertificate({
       setLitterError(true);
       error = true;
     }
+    if (!individual?.litter6w) {
```

Comment:
There's a mistake here, it should be 
```suggestion
    if (!individual?.litter_size6w) {
```

Reply:
yes